<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Найти</string>
    </attribute>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FindAllButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти все</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FindNextButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать далее</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="WordCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>99</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Слово целиком</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CaseCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>78</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Регистр</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Направление</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>15</y>
        <width>82</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Вверх</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>39</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Вниз</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ChanelSearchButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Заменить</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>10</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Заменить:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReplaceButton">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Заменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FindButton">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>10</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReplaceAllButton">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Заменить все</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Заменить</string>
     </property>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Вкладки</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Файлы</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ChanelReplaceButton">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>100</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>99</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Регистр</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>78</y>
       <width>99</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Слово целиком</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="replaceLine">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>40</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
